### PR TITLE
event transform and rotation

### LINKED ISSUE #75: Rotate text
It would nice to have a method to draw text at any angle. Some UIs have text running along the side, like left tabs.   
I'm not sure if cosmic text supports this automatically?

### PR DESCRIPTION
**Pull request by jrmoulton**
Fixes #75

### CODE REVIEW

**Comment by jrmoulton** [comment]
https://github.com/user-attachments/assets/39a77226-65e4-4e85-9566-bdc86d02bd7c

**Comment by jrmoulton** [comment]
@timsueberkrueb finally got around to this

File: tiny_skia/src/lib.rs [hunk 1]
```diff
@@ -237,9 +237,19 @@ impl<W> TinySkiaRenderer<W> {
             ..Default::default()
         };
 
+        let transform = transform.as_coeffs();
+        let scale = self.scale as f32;
+        let transform = Transform::from_row(
```
Comment (timsueberkrueb) [inline tiny_skia/src/lib.rs:242]: Why is this needed?

**Comment by timsueberkrueb** [comment]
This is great, thank you so much!
